src/ripple/app/misc/SHAMapStoreImp.cpp

```diff
@@ -269,6 +272,7 @@ void
 SHAMapStoreImp::run()
 {
     LedgerIndex lastRotated = state_db_.getState().lastRotated;
+    Ledger::pointer validatedLedger;
```

Comment:
Damn, and I thought I was so clever to notice that it didn't need to be a member variable. :wink: 

Fixed.